src/Language/PureScript/Constants.hs

```diff
@@ -457,6 +460,12 @@ pattern SymbolAppend = Qualified (Just PrimSymbol) (ProperName "Append")
 pattern SymbolCons :: Qualified (ProperName 'ClassName)
 pattern SymbolCons = Qualified (Just PrimSymbol) (ProperName "Cons")
 
+pattern SymbolContains :: Qualified (ProperName 'ClassName)
+pattern SymbolContains = Qualified (Just PrimSymbol) (ProperName "Contains")
+
+pattern SymbolBreakOn :: Qualified (ProperName 'ClassName)
+pattern SymbolBreakOn = Qualified (Just PrimSymbol) (ProperName "BreakOn")
```

Comment:
This isn't supposed to be in this PR is it?

Reply:
oh, rebase failure